bin/pack-resources.js

```diff
@@ -67,9 +67,20 @@ for (const dirent of await readdir(wasmDirectory, { withFileTypes: true })) {
 output += `\
 });
 
-const filesystem = async (fetch) => ({
+const filesystem = async (fetch) => {
+    var chunks = [];
 `;
 
+function chunks(data, length) {
+    var rest = data;
+    var chunks = [];
+    while(rest.length != 0) {
+        chunks.push(rest.subarray(0, length));
+        rest = rest.subarray(length);
+    }
+    return chunks;
+}
+
```

Comment:
I think this is a little nicer?

```suggestion
function* chunks(data, length) {
    while (data.length != 0) {
        yield data.subarray(0, length);
        data = data.subarray(length);
    }
}

```

Reply:
If you say so, javascript is not my native language...